tachyon/zk/plonk/permutation/unpermuted_table.h

```diff
@@ -29,8 +29,8 @@ class UnpermutedTable {
  public:
   using F = typename PCSTy::Field;
   using Domain = typename PCSTy::Domain;
-  using Rows = std::vector<F>;
-  using Table = std::vector<Rows>;
+  using Cols = std::vector<F>;
```

Comment:
I think `Col` or `Column` is more appropriate than 'Cols' for `std::vector<F>`, as it represents a single column in a table.